documentation/yaml/sdk.yaml

```diff
@@ -0,0 +1,198 @@
+openapi: 3.0.0
+servers:
+- url: https://demo.dpppt.org
+  description: ''
+info:
+  version: 1.0-gapple
+  description: DP3T API
+  title: DP3T API
+paths:
+  /v1/:
+    get:
+      summary: hello
+      description: Echo endpoint
+      responses:
+        '200':
+          description: Hello from DP3T Config WS
+          content:
+            application/json:
+              schema:
+                type: string
+  /v1/config:
+    get:
+      summary: getConfig
+      description: Read latest configuration and messages, depending on the version
+        of the phone and the app.
+      responses:
+        '200':
+          description: ConfigResponse structure with eventual notifications and epidemic
+            parameters
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/org.dpppt.switzerland.backend.sdk.config.ws.model.ConfigResponse'
+      parameters:
+      - name: osversion
```

Comment:
Seems to work, before we had here 'arg0' :)

Reply:
yes, looks nicer!